### DISCUSSION TITLE
onBackPressed and Android 13

### DISCUSSION THREAD

**Discussion by casolorz** [AppIntro/AppIntro#1017] [General]
Today I got an email from Google saying I can no longer override `onBackPressed` on Android 13, so I searched my code to see where all I use it and the only place I found it outside my code (I use it in my code as well) was in this library. 

Are there any plans to update that?

Thanks.

**Comment by cortinico** [comment]
> Today I got an email from Google saying I can no longer override `onBackPressed` on Android 13

Could you provide more reference to this?

**Reply by casolorz** [reply]
They basically sent on of their typical Play Store compliance emails that said this:

`We've identified that your Android app(s) potentially uses unsupported KEYCODE_BACK or onBackPressed APIs, which are no longer supported in Android 13.`

And they gave me this link https://developer.android.com/about/versions/13/features/predictive-back-gesture

I just got it earlier today so I haven't had time yet to go analyze the issue in detail, I just wanted to see how easily it'll be to fix the code everywhere I use it.

Thanks.

Edit: it looks like a policy email but it is from the compatibility team so it might not be a policy issue.

**Reply by cortinico** [reply]
Yup it seems like we'll have to update our back logic handling to accomodate for those apis otherwise users won't be able to use 'predictive back gesture'
